src/platforms/dotnet/common/configuration/msbuild.mdx

```diff
@@ -83,9 +83,26 @@ You can add Sentry's MSBuild properties to any property group. For example:
 
 ```xml
 <PropertyGroup>
+
+  <!-- Configure Sentry org and project -->
   <SentryOrg>___ORG_SLUG___</SentryOrg>
   <SentryProject>___PROJECT_SLUG___</SentryProject>
+
+  <!--
+    Optionally, you can upload sources to Sentry - enabling Source Context.
+    Note, symbols are sent by default for Release configuration, but sending sources is always an opt-in feature.
+  -->
+  <SentryUploadSources Condition="'$(Configuration)' == 'Release'">true</SentryUploadSources>
```

Comment:
I thought we were internally only uploading in Release already. An additional `<SentryUploadSourceDebug>true</SentryUploadSourceDebug>` which a nice to have, since Sentry is really a release/prod thing for the most part.